splinterd/src/node/runnable/mod.rs

```diff
@@ -20,6 +20,7 @@ pub(super) mod network;
 use std::net::{Ipv4Addr, SocketAddr};
 
 use splinter::error::InternalError;
+use splinter::events::Reactor;
```

Comment:
This has actix requirements and should be handled separately based on actix web 1 vs 3.

Reply:
I've added  AdminServiceEventClientVariant enums, which acts similarly to the RestApiVariant enums